lib/server.js

```diff
@@ -3,7 +3,7 @@ import { server as baseServer, routeConfiguringFunction  } from 'appium-base-dri
 import { WindowsDriver } from './driver';
 
 async function startServer(port, host) {
-    let driver = new WindowsDriver(port, host);
+    let driver = new WindowsDriver({port, host});
```

Comment:
This should probably be `address` instead of `host`, as that is what the appium driver will pass in when it instantiates the driver on session creation.